cli/actions/setup.go

```diff
@@ -0,0 +1,40 @@
+package actions
+
+import (
+	"path/filepath"
+
+	"github.com/NethermindEth/sedge/configs"
+	"github.com/NethermindEth/sedge/internal/pkg/commands"
+	log "github.com/sirupsen/logrus"
+)
+
+type SetupContainersOptions struct {
+	GenerationPath string
+	Services       []string
+}
+
+func (s *sedgeActions) SetupContainers(options SetupContainersOptions) error {
```

Comment:
Log debug the three commands

Reply:
Done 0b0efbfd5e6bc5a5ddaf23a162f0e761610ada2a